src/components/SearchBar.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+
+const SearchBar = () => {
+    const [searchTerm, setSearchTerm] = useState("");
+
+    function onSearch(e) {
+        e.preventDefault()
+        const form = document.querySelector('#search-form');
+        const data = new FormData(form);
+        setSearchTerm(data.get("searchQuery"))
```

Comment:
I think it is okay  to use querySelector for this moment, but in projects common practice is using  formik or react-forms library to handle form interactions

Also to get element value we can use useRef() hook
`

    const ref = useRef();

    const onSearch = (e) => {
        e.preventDefault()
        setSearchTerm(ref.current.value)
    }

    return (
        <form onSubmit={onSearch} >
            <input ref={ref} />
            <button>Search</button>
            <h1>{searchTerm}</h1>
        </form>
    );
}

`

Reply:
@MykhailoSheliahov updated in module3